src/flint/fmpq.jl

```diff
@@ -138,6 +138,20 @@ function deepcopy_internal(a::fmpq, dict::IdDict)
    return z
 end
 
+@doc Markdown.doc"""
+    floor(a::fmpq)
+> Returns the greatest integer that is less than or equal to $a$. The result is
+> returned as a rational with denominator $1$.
+"""
+Base.floor(a::fmpq) = fmpq(fdiv(numerator(a), denominator(a)), 1)
+
+@doc Markdown.doc""";
+    floor(a::fmpq)
```

Comment:
Adjust definition for `ceil`

Reply:
Done.

Reply:
It seems you didn't update the signature in the docstring though!
```suggestion
    ceil(a::fmpq)
```

Reply:
Whoops, done. Thanks!